### PR TITLE
Added Roll Definitions to Glossary

### PR DESCRIPTION
**Pull request by Reveraine**
Average rolling was something I thought of while reworking races. I don't think it's too complex, but it will need some testing, I think. I'll put more thoughts for it in basic rules.

### CODE REVIEW

File: Glossary_of_Terms.txt
Comment (bleehu) [inline Glossary_of_Terms.txt]: Should we add a "See the Primary Game Mechanic section of the Basic Rules" note in the description of each of these rolls? Not a blocking issue.

**Review by bleehu — APPROVED** [review]
One question, not blocking. Ship it if you'd like!

Rev, Way to go making your first few pulls! Just a reminder since this is your first couple of times that you get to push the big green merge button-of-win! Don't forget to delete the branch after it's been merged to keep the list of branches that are in use short. Way to go!!!

File: Glossary_of_Terms.txt
Comment (Reveraine) [reply Glossary_of_Terms.txt]: That could be nice. I can do that.

**Pull request merged**
